operate/client/e2e-playwright/tests/processInstance.spec.ts

```diff
@@ -199,8 +198,8 @@ test.describe('Process Instance', () => {
     ).toBeVisible();
   });
 
-  test('Cancel an instance', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```

Comment:
I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)